pkg/controller/workload/reconciler.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
 	"github.com/vmware-tanzu/cartographer/pkg/conditions"
-	"github.com/vmware-tanzu/cartographer/pkg/realizer"
+	realizerworkload "github.com/vmware-tanzu/cartographer/pkg/realizer/workload"
```

Comment:
I noticed that under each package we're naming the import after the name of the controller

I wonder if that's a smell that the way we broke the modules down are not after the domain they're working on, but instead, the "thing they are"

regardless, I'd suggest keeping

```suggestion
	realizer "github.com/vmware-tanzu/cartographer/pkg/realizer/workload"
```

but no strong opinions really

Reply:
We definitely have some work/refactoring to do in the realizer. I have changed all the import names except in registrar.go